tests/security/yast2_apparmor/scan_audit_logs.pm

```diff
@@ -48,8 +50,13 @@ sub run {
     # Setup testing profile/auditlog
     # E.g., comment out specific entries from profile, then run corresponding programs to generate audit records
     assert_script_run("cp $test_profile $test_profile_bk");
-    assert_script_run("sed -i -e 's/$entry/#$entry/' $test_profile");
-    validate_script_output("grep '##' $test_profile", sub { m/#$entry/ });
+    if (is_sle) {
+        assert_script_run("sed -i -e 's/$entry/#$entry/' $test_profile");
+        validate_script_output("grep '##' $test_profile", sub { m/#$entry/ });
```

Comment:
Why added a new '#' here? will be no impact if we don't add it, correct?

Reply:
Correct, I also do think it makes no difference in SLE between using '##' or '#', but I just kept this part of code as its original way :)

Reply:
How about use this format?
my $entry = 'include <abstractions\/base>';
 assert_script_run("sed -i -e 's/$entry/#$entry/' $test_profile");

Reply:
I guess it should work, that way we can merge two assert_script_run into one but we still need two seperate validate_script_output() for tw and sle, right? SInce there will still be '##' in sle and '#' in tw.

Reply:
hmm, match only one "#" can't work?